### ISSUE TITLE
gunicorn 20.0.2 wheel contains extra files

### ISSUE THREAD

**Issue by a-feld** [benoitc/gunicorn#2204]
Hi! Upon inspecting the [gunicorn 20.0.2 wheel](https://files.pythonhosted.org/packages/4c/f4/1fff9549bda65801f34bd02a86bde95c945401bfc21cc9cb9e35ec790ba5/gunicorn-20.0.2-py2.py3-none-any.whl) we noticed there are extra files that seem to have snuck in.

SHA256 hash of wheel: `fe457b7093b353cefcb2ebc40d192fc7c8a62bba49da648c9e1fa0671fa654d1`

```
$ diff -r wheel/ tarball/
Only in wheel/: _compat.py
Only in wheel/: argparse_compat.py
Only in wheel/http: _sendfile.py
Only in wheel/: selectors.py
Only in wheel/: six.py
Only in wheel/workers: _gaiohttp.py
Only in wheel/workers: gaiohttp.py
```

**Comment by benoitc** [comment]
hrm odd... I will fix it in the coming 20.0.3. Thanks for the report :)

**Comment by benoitc** [comment]
20.0.3 is out.
